ui/src/views/AutogenView.vue

```diff
@@ -621,6 +622,9 @@ export default {
     next()
   },
   beforeRouteLeave (to, from, next) {
+    console.log('DEBUG - Due to route change, cancelling any on-going API request', this.apiName)
```

Comment:
@rohityadavcloud when it says cancelling on-going API request, we are just canceling the response handling of the API call?

Reply:
thanks @shwstppr , that was confusing me.

Reply:
```suggestion
    console.log('DEBUG - Due to route change, ignoring results for any on-going API request', this.apiName)
```

Reply:
The log doesn't matter, I'll remove it before it gets merged; see PR description `Add console log for debugging which may be removed if this passes code review and testing
`